modules/Bio/EnsEMBL/Compara/Utils/Test.pm

```diff
@@ -278,6 +298,30 @@ sub load_statements {
 }
 
 
+=head2 get_pipeline_tables_create_statements
```

Comment:
Would be good to update `t/Utils/Test.t` to add tests for these new methods

Reply:
I have added a test for `get_pipeline_tables_create_statements` (which calls `read_sqls`). `apply_statements` (like `test_schema_compliance` or `test_command`) can't easily be tested because they're wrappers around `fail` & `pass` from `Test::More`